### PR TITLE
Minor fixes to the sources and requirements

### PR DESCRIPTION
**Pull request by Tiboris**
Fixing:
db_driver.load() to take no params as the path is handled in init
ctx object should have ctx.obj to be accessed with key
Adding:
requirements were missing when installing in clear environment
yaml lib helper functions

### CODE REVIEW

File: src/aiohabit/run.py [hunk 1]
```diff
@@ -63,8 +63,8 @@ def aiohabitcli(ctx, config, db):
     """Multihost human friendly provisioner."""
     init_providers()
     ctx.ensure_object(dict)
-    ctx[DB] = init_db(db)
-    ctx[CONFIG] = init_config(config)
+    setattr(ctx, DB, init_db(db))
```
Comment (pvoborni) [inline src/aiohabit/run.py:66]: In other project I've fixed this issue by:

```python
@click.pass_context
def managercli(ctx, config, verbose):
    """Uploads new OpenStack images from OS composes"""
    ctx.ensure_object(dict)
    ctx.obj[CONFIG] = init_config(config)
```

Comment (Tiboris) [reply src/aiohabit/run.py:66]: So what what would be our preference?

Comment (pvoborni) [reply src/aiohabit/run.py:66]: The method which I wrote. It is also used in the official documentation: https://click.palletsprojects.com/en/7.x/commands/#nested-handling-and-contexts

Comment (pvoborni) [reply src/aiohabit/run.py:66]: I.e. my initial implementation in aiohabit simply forgot the `obj` part.

Comment (Tiboris) [reply src/aiohabit/run.py:66]: Let me update it.

**Pull request merged**